man/kpatch.1

```diff
@@ -36,7 +36,10 @@ list
        list installed patch modules
 
 signal
-       signal/poke any process stalling the current patch transition
+       signal/poke any process stalling the current patch transition.
+       This is only usefull on systems that have the sysfs livepatch signal interface.
```

Comment:
spelling nit: s/usefull/useful

Reply:
Ah, good catch, I'll update that.
Thanks.